server/models/tripModel.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+var ObjectId = Schema.Types.ObjectId;
+
+const TripSchema = new Schema({
+  user: String,
+  date: { type: Date, default: Date.now },
+  comments: [
+    {
+      user: ObjectId,
+      body: String,
+      date: Date
+    }
+  ],
+  stops: [
+    {
+      index: Number,
+      coordinates: [String],
+      next_stop: { type: Number, default: null },
+      previous_stop: Number,
```

Comment:
Please define the default value in this prop as well, the default value should be **null**

Reply:
Got it, thanks